presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -31,24 +31,10 @@ Configuration Properties
 
 The following configuration properties are available:
 
-============================================= ==============================================================================
```

Comment:
I removed the tables since they duplicate content. Alternatively we could keep the tables (maybe convert them to list tables) and remove the lists with the titles. I opted for removing the tables since the current setup has each property in the nav on the right .. which is quite nice